utop.opam

```diff
@@ -22,4 +22,4 @@ depends: [
   "jbuilder"     {build & >= "1.0+beta9"}
 ]
 build-test: [["jbuilder" "runtest" "-p" name "-j" jobs]]
-available: [ocaml-version >= "4.02.3"]
+available: [ocaml-version >= "4.03.0"]
```

Comment:
Why this change?

Reply:
String.lowercase_ascii/captialize_ascii are since 4.03.0